### PR TITLE
Simple chart organism

### PR DESCRIPTION
**Pull request by wpears**
Introduces the simple-chart organism, currently available to be added to browse pages.

Similar to CCT (and the chart organism), data-* attributes are created in the jinja2 template which are then accessed in the javascript the is used to build the charts.

The main interesting bits of customization are the `Transform` field and the `Style Overrides` field in wagtail. These are used as follows:

`Transform`: provide a function name that is added to `chart-hooks.js` as a method on the exported object. This function is run on the provided `data-source` and passes the transformed data to the chart.
`Style Overrides`: A JSON object with Highchart customizations (see https://api.highcharts.com/highstock/). To reference nested objects, you can provide keys with dot notation, eg `"tooltip.formatter"`. You can provide normal JSON datatypes, but also functions and *results* of functions. To provide a function (eg, `tooltip.formatter` expects a function), reference a method in `chart-hooks.js` prefixed with `fn__`, eg `"fn__enforcement_barTooltipFormatter"`. To provide the *results* of a function, reference a method in `chart-hooks.js`, prefixed with `hook__`, eg `"hook__enforcement_barCategories"`.

Screen Shot 2020-11-13 at 2 51 47 PM

The `scripts/` are temporary, to be removed when we finish the enforcement data API in the coming weeks.

### CODE REVIEW

File: cfgov/v1/atomic_elements/organisms.py [hunk 1]
```diff
@@ -588,6 +588,89 @@ def get_limit(self, value):
...
+        default='datetime',
+        required=True
+    )
+
+    data_source = blocks.TextBlock(
+        required=True,
+        help_text='URL of the chart\'s data source or an array of JSON data'
+    )
+
+    description = blocks.CharBlock(
```
Comment (schbetsy) [inline cfgov/v1/atomic_elements/organisms.py:611]: Where is this used? When I inspect a page with a chart, I don't see this description in the source.

**Review by schbetsy — APPROVED** [review]
I didn't do a close review of the JS and less code in this PR, since it's gotten pretty extensive review by UX and graphic designers on our team. I added some comments about documentation and help text in Wagtail, since we're hoping for this block to be useful in many contexts. In general, this looks great, and I suspect we'll be able to merge today.

File: cfgov/v1/atomic_elements/organisms.py [hunk 2]
```diff
@@ -588,6 +588,89 @@ def get_limit(self, value):
...
+
+    transform = blocks.CharBlock(
+        required=False,
+        help_text='Name of the javascript function in chart-hooks.js to run '
+        'on the provided data before handing it to the chart'
+    )
+
+    style_overrides = blocks.TextBlock(
+        required=False,
+        help_text='A JSON object with style overrides for the underlying '
```
[the remark this answers is not in the document]
Comment (wpears) [reply cfgov/v1/atomic_elements/organisms.py:634]: Agreed. Task created at https://github.local/CFPB/super-regular/issues/208

File: cfgov/v1/atomic_elements/organisms.py (hunk 1 above)
Comment (wpears) [reply cfgov/v1/atomic_elements/organisms.py:611]: This was a bug! Fixed in 30b1bd56645e9f5948c8f8d04839408b6a91299b

File: cfgov/scripts/parse_counts.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import csv
+from datetime import datetime as dt
+
+
```
[the remark this answers is not in the document]
Comment (wpears) [reply cfgov/scripts/parse_counts.py:8]: Done in 30b1bd566

**Pull request merged**